### ISSUE TITLE
Change DNS Settings for netifly

### ISSUE THREAD

**Issue by dvidsilva** [techqueria/website#9]
https://app.netlify.com/sites/techqueriawebsite/overview

Is blocked by others, but keeping it here for reference

**Comment by fvcproductions** [comment]
Once #11 is configured correctly for everything, we can go ahead and change DNS settings within Netlify! 👍

**Comment by fvcproductions** [comment]
@dvidsilva Were we thinking of changing the template first before doing this step or how should we go about this?

**Comment by dvidsilva** [comment]
@fvcproductions  yes i was thinking that. maybe in the meantime we can make a dns setting for a subdomain, like new.techqueria.org or beta, idk, let me know what you think and i can adjust that.

**Comment by dvidsilva** [comment]
@fvcproductions  Done! sorry the delay, http://beta.techqueria.org/ with this we can probably now invite other folks to collaborate, and give us ideas for design, templates, content, etc. I can make an announcement, or you can do it, maybe on the #random channel.

**Comment by dvidsilva** [comment]
We'll open a new issue for when ready. But closing this one for know!
